method/index.js

```diff
@@ -1,19 +1,52 @@
+const Crut = require('ssb-crut')
 const Init = require('./group/init')
 const AddMember = require('./group/add-member')
-const CreateLink = require('./link/create')
 const FindByGroupByFeedId = require('./link/find-group-by-feedid')
 const Application = require('./application')
+const FeedGroupLink = require('../spec/link/feed-group')
+const GroupSubgroupLink = require('../spec/link/group-subgroup')
 
 module.exports = function Method (ssb, keystore, state) {
   const application = Application(ssb)
+  const linkFeedCrut = new Crut(ssb, FeedGroupLink)
+  const linkSubgroupCrut = new Crut(ssb, GroupSubgroupLink)
 
   return {
     group: {
       init: patient(Init(ssb, keystore, state)),
       addMember: patient(AddMember(ssb, keystore, state))
     },
     link: {
-      create: patient(CreateLink(ssb)),
+      create ({ group, name }, cb) {
+        const input = {
+          parent: ssb.id,
+          child: group,
+          name,
+          recps: [group]
+        }
+
+        // set the recps on the link to be the same as the recps on the settings
+        if (!name) delete input.name
+
+        patient(linkFeedCrut.create.bind(linkFeedCrut)(input, (err, linkId) => {
```

Comment:
:fire: can't remember how `patient` works, but I don't this this will work exactly. as you've passed in a function that is being called directly

Reply:
i.e. 
```
function add (a, b) { return a + b }

patient(add)(2, 3)  // waits on patient before calling add
patient(add(2, 3)) // runs add immediately

Reply:
How should i call it?